modules/about-using-gpu-operator.adoc

```diff
@@ -1,21 +1,87 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
-
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-virtual-gpus.adoc
 
 :_content-type: CONCEPT
 [id="about-using-nvidia-gpu_{context}"]
 = About using the NVIDIA GPU Operator
 
-The NVIDIA GPU Operator manages NVIDIA GPU resources in a {product-title} cluster and automates tasks related to bootstrapping GPU nodes. Because the GPU is a special resource in the cluster, you must install some components before you can deploy application workloads to the GPU. These components include the NVIDIA drivers that enable the compute unified device architecture (CUDA), Kubernetes device plugin, container runtime, and other features such as automatic node labeling, monitoring, and more.
+You can use the NVIDIA GPU Operator with {VirtProductName} to rapidly provision worker nodes for running GPU-enabled virtual machines (VMs). The NVIDIA GPU Operator manages NVIDIA GPU resources in an {product-title} cluster and automates tasks that are required when preparing nodes for GPU workloads.
+
+Before you can deploy application workloads to a GPU resource, you must install components such as the NVIDIA drivers that enable the compute unified device architecture (CUDA), Kubernetes device plugin, container runtime, and other features, such as automatic node labeling and monitoring. By automating these tasks, you can quickly scale the GPU capacity of your infrastructure. The NVIDIA GPU Operator can especially facilitate provisioning complex artificial intelligence/machine learning (AI/ML) workloads.
 
-[NOTE]
-====
-The NVIDIA GPU Operator is supported only by NVIDIA. For more information about obtaining support from NVIDIA, see link:https://access.redhat.com/solutions/5174941[Obtaining Support from NVIDIA].
-====
+[id="options-configuring-gpu-operator_{context}"]
+== Options for configuring the GPU Operator
 
-There are two ways to enable GPUs with {product-title} {VirtProductName}: the {product-title}-native way described here and by using the NVIDIA GPU Operator.
+There are two available methods for configuring the NVIDIA GPU Operator.
```

Comment:
`There are two available methods for configuring mediated devices using the NVIDIA GPU Operator`